.vscode/settings.json

```diff
@@ -55,14 +55,19 @@
     "Plotly",
     "Unmount",
     "automl",
+    "bdist",
```

Comment:
what is this file used for?

Reply:
for spell checking, in extension.json, we have an extension for spell checking